modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -57,7 +61,13 @@ object Renderer {
       Result(unit.namespace, decl.name, content)
     }
 
-    pack :: classes
+    val packageApplicableDecls = unit.declarations.filter {
+      case _: TypeAlias | _: Service => true
+      case _                         => false
+    }
+
+    if (packageApplicableDecls.isEmpty) classes
+    else pack :: classes
```

Comment:
This may not be the preferred way to do this, but without something like this an empty package object is created for the `import_test` package